Sources/StreamChat/Database/DTOs/CurrentUserDTO.swift

```diff
@@ -203,15 +203,16 @@ extension CurrentChatUser {
     fileprivate static func create(fromDTO dto: CurrentUserDTO) throws -> CurrentChatUser {
         let user = dto.user
 
-        let extraData: [String: RawJSON]
-        do {
-            extraData = try JSONDecoder.default.decode([String: RawJSON].self, from: dto.user.extraData)
-        } catch {
-            log.error(
-                "Failed to decode extra data for user with id: <\(dto.user.id)>, using default value instead. "
-                    + "Error: \(error)"
-            )
-            extraData = [:]
+        var extraData = [String: RawJSON]()
+        if !dto.user.extraData.isEmpty {
```

Comment:
If data is empty, error is logged which can happen when trying to convert deleted DTO